CppCoreGuidelines.md

```diff
@@ -10385,7 +10385,7 @@ Readability and safety.
 
 As an optimization, you may want to reuse a buffer as a scratch pad, but even then prefer to limit the variable's scope as much as possible and be careful not to cause bugs from data left in a recycled buffer as this is a common source of security bugs.
 
-    {
+    void writeFile() {
```

Comment:
Based on a quick grep, looks like `snake_case` would be better here for consistency.